x/cdp/keeper/params.go

```diff
@@ -79,3 +78,19 @@ func (k Keeper) getLiquidationRatio(ctx sdk.Context, denom string) sdk.Dec {
 	}
 	return cp.LiquidationRatio
 }
+
+func (k Keeper) getLiquidationPenalty(ctx sdk.Context, denom string) sdk.Dec {
+	cp, found := k.GetCollateral(ctx, denom)
+	if !found {
+		panic(fmt.Sprintf("collateral not found: %s", denom))
```

Comment:
Will panic if collateral type type 'random' is removed in between CDP deposit of 'random' and liquidation of the CDP.

Reply:
Will file an issue on this topic. It's worth documenting and thinking about how we want to handle it, but for now, it would be unlikely for governance to accept a proposal to remove a collateral that still had active cdps.